scripts/loki_transform.py

```diff
@@ -417,14 +419,18 @@ def convert(
     mode = mode.replace('-', '_')  # Sanitize mode string
     if mode in ['c', 'cuda_parametrise', 'cuda_hoist']:
         if mode == 'c':
-            f2c_transformation = FortranCTransformation(path=build)
+            f2c_transformation = FortranCTransformation()
+            f2c_wrapper = FortranISOCWrapperTransformation()
         elif mode in ['cuda_parametrise', 'cuda_hoist']:
-            f2c_transformation = FortranCTransformation(path=build, language='cuda', use_c_ptr=True)
+            f2c_transformation = FortranCTransformation(language='cuda', use_c_ptr=True)
```

Comment:
```suggestion
            f2c_transformation = FortranCTransformation(language='cuda')
```

There is no `use_c_ptr` argument for `FortranCTransformation` anymore.